src/gossip/gossip.lisp

```diff
@@ -58,6 +58,8 @@
 
 (defvar *debug-level* 1 "True to log debugging information while handling gossip requests. Larger values log more messages.")
 
+(defvar *transport* nil "Network transport endpoint.")
```

Comment:
It would be better if we could somehow not have to touch `gossip.lisp` to boot the transport code, so that `gossip/transport` could be tested independently of `gossip`, but trying to hack this separation doesn't look trivial.